README.rst

```diff
@@ -256,3 +256,77 @@ data that the node is no longer responsible for.
 .. _Jolokia: https://jolokia.org/
 .. _STUPS Cassandra: https://github.com/zalando/stups-cassandra
 .. _Più: http://docs.stups.io/en/latest/components/piu.html
+
+Upgrade your cluster from Cassandra 2.1 -> 3.x
+===================
+
+In order to upgrade your Cluster you should run the following steps. You should have in mind that this process is a rolling update, which means applying the changes for each node in your cluster one by one.
+After upgrading the last node in your cluster you are done.
+
+**Disclaimer: Before you actually start, you should:**
+  1. Read the [Datastax guide](https://docs.datastax.com/en/latest-upgrade/upgrade/cassandra/upgrdCassandraDetails.html) and consider the upgrade restrictions.
+  2. Check if your client applications driver actually support V4 of the cql-protocol
+
+
+1. Check for the latest Plan-B Cassandra image version: 
+  `curl https://registry.opensource.zalan.do/teams/stups/artifacts/planb-cassandra-3/tags | jq '.[-1].name'`
+2. Connect to the instance where you want to run the upgrade and enter your docker container. 
+3. Run `nodetool upgradesstables` and `nodetool drain`. The latter command will flush the memtables and speed up the upgrade process later on. *This command is mandatory and cannot be skipped.*
+   Excerpt from the manual `Cassandra stops listening for connections from the client and other nodes. You need to restart Cassandra after running nodetool drain.`
+4. Stop the docker container and remove it
+  ```
+    Inside the container:
+      nodetool stopdaemon
+    On the host:
+      docker stop taupageapp
+      docker rm taupageapp
+  ```
```

Comment:
Well, after the drain, you can just do `docker rm -f taupageapp`.  There's no need to stop the container after you've stopped cassandra process, the container will stop itself.  And since the node is drained, there's no need to stop the daemon, killing it is good enough.